pdf/plugin-dowker/index.pug

```diff
@@ -0,0 +1,79 @@
+!!! 5
```

Comment:
At the moment, it looks like you aren't using this template, so wouldn't need to include it. However, I'll point out that the reason this template exists with the Bernoulli test plugin is so that the FAW-specific data can be injected into the HTML which gets sent to the user. If your compiled HTML file can be similarly decomposed into a template like this, that might be a good approach.